src/data/polynomial.lean

```diff
@@ -1628,6 +1750,10 @@ eval₂_neg _
 @[simp] lemma eval_sub (p q : polynomial R) (x : R) : (p - q).eval x = p.eval x - q.eval x :=
 eval₂_sub _
 
+lemma coeff_mul_X_sub_monomial {p : polynomial R} {r : R} {a : ℕ} :
```

Comment:
Hmm, shouldn't this be `X_sub_C`? This doesn't seem the "normal form"

Reply:
(Just a general request to reviewers --- if you click on drag to select several lines, then you can see the whole theorem statement even in the "conversation view" in github. If you only comment on the top line, there's insufficient context to understand the comment without going to the "files" tab.)

Reply:
I've previously been unconvinced we should have a norm form preference between `monomial 0` and `C`. I'll try this simp lemma now, in any case, and see how it goes.

Reply:
Okay, I've added the `@[simp]` lemma from `monomial 0` to `C`, and changed all the lemmas appropriately. It is indeed nicer.